PGM/src/main/java/tc/oc/pgm/polls/types/PollCustom.java

```diff
@@ -0,0 +1,37 @@
+package tc.oc.pgm.polls.types;
+
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.assistedinject.AssistedInject;
+import org.bukkit.command.CommandSender;
+import tc.oc.commons.core.chat.Audiences;
+import tc.oc.pgm.polls.Poll;
+import tc.oc.pgm.polls.PollManager;
+
+public class PollCustom extends Poll {
+
+    public interface Factory {
+        PollCustom create(String text, CommandSender initiator);
+    }
+
+    private String text;
+
+    @AssistedInject PollCustom(@Assisted String text, @Assisted CommandSender initiator, PollManager pollManager, Audiences audiences) {
```

Comment:
My problem is,  @AssistedInject is used to indicate that there are multiple constructors. I think @Inject would be correct here instead.

Reply:
The `Poll` class already defines an `Inject`ed constructor, I believe `AssistedInject` is correct here